monitoring reverse proxies (#438)
uyuni-project/uyuni-docs: modules/administration/pages/monitoring.adoc

Monitoring formulas can also be configured for System Groups, by applying the same configuration used for individual systems inside the corresponding group.
====

> // The WebUI says that we describe the available arguments in the exporters formula, but I don't see it. Might be best to put it in the Salt Guide, then xref from here. --LKB 2020-08-11

Reviewer: The help string says: _Please refer to the documentation for available options._. These are different for every exporter. I'm not sure if that's in scope of product documentation. @renner , could you comment on this please?

Author: Let's address this in https://github.com/SUSE/spacewalk/issues/12001 so we can merge this one now.

Tools like PushProx deploy a proxy and a client on both sides of the network barrier and allow Prometheus to work across network topologies such as NAT.

For more information on PushProx, see https://github.com/RobustPerception/PushProx.

> === Reverse Proxy Set Up

Reviewer 2: ```suggestion
=== Reverse Proxy Setup
```

Author: Heh, good spot. In my defence, I originally had "Set Up Reverse Proxy" :wink:

Prometheus fetches metrics using a pull mechanism, so the server must be able to establish TCP connections to monitored clients.
If you cannot open these ports directly in your firewall, you can use reverse proxies to establish a connection.

// Probably a diagram here. --LKB 2020-08-11

.Procedure: Installing Prometheus with Reverse Proxies

> . In the {productname} {webui}, open the details page of the system where Prometheus is to be installed, and navigate to the [guimenu]``Formulas`` tab.

Reviewer: ```suggestion
. In the {productname} {webui}, open the details page of the system to be monitored, and navigate to the [guimenu]``Formulas`` tab.
```
